### ISSUE TITLE
Importing feeds sometimes imports only one feed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#18]
```
What steps will reproduce the problem?
1. Add and update a feed
2. Import a feed.

What is the expected output?  Feeds from the input
What do you see instead?  The last feed replaces the feed that was updated.
Need to set current index to -1, to prevent this.

This was fixed in version 1.8.
```

Original issue reported on code.google.com by `ibunto...@gmail.com` on 14 Aug 2007 at 8:21

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 1.9.
```

Original comment by `ibunto...@gmail.com` on 6 Sep 2007 at 1:38
- Added labels: **Milestone-Release1.9**, **Type-Patch**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Tested OK
```

Original comment by `ibunto...@gmail.com` on 6 Sep 2008 at 2:52
- Changed state: **Verified**
